assets/src/edit-story/utils/useIntersectionEffect.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+const IntersectionObserver = require('intersection-observer-polyfill/dist/IntersectionObserver');
```

Comment:
I don't love this. Did you ever workout why this is happening? Can you explain. If it can't be helped, a comment should be added.

Reply:
Yes, it's up to the library author (build config). I hope we will remove all polyfills after we agree on supported browsers.